eldritch/test_mythos.py

```diff
@@ -2627,6 +2653,63 @@ def testUndeadIgnoredByRedSign(self):
     self.assertEqual(vampire.toughness(self.state, self.char), 3)
 
 
+class Mythos57Test(EventTest):
+  def setUp(self):
+    super().setUp()
+    self.whiskey0 = items.Whiskey(0)
+    self.char.possessions.append(self.whiskey0)
+    self.state.mythos.appendleft(Mythos57())
+
+  def tearDown(self):
+    self.assertEqual(self.state.places["Roadhouse"].closed_until, 2)
+
+  def testLoseWhiskeyWhenArrested(self):
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=1)):
+      self.advance_turn(1, "movement")
+    self.assertFalse(self.char.possessions)
+    self.assertEqual(self.char.place.name, "Police")
+    self.assertEqual(self.char.arrested_until, 2)
+
+  def testMultipleWhiskeyOneCheck(self):
+    self.char.speed_sneak_slider -= 1
+    whiskey1 = items.Whiskey(1)
```

Comment:
Might make more sense to add this whiskey to the first test case, to make sure both whiskeys get discarded.